vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -50,7 +50,7 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
-      new_result.merge(
+      new_result.merge!(
```

Comment:
> @abonas, it's a good point ... usually when I hit a line like this in the debugger, I'll execute the line directly in the debugger to see the return value, as long as I can expect that there are no side effects.

yeah, that's called "evaluate expression" in IDE, but as you mentioned, sometimes code can't be ran twice in the same scope for various reasons.